toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -950,21 +950,21 @@ popd
 rm -rf texinfo-6.8
 touch /logs/status_texinfo_complete
 
-echo Procps-ng-3.3.15
-tar xf procps-ng-3.3.15.tar.xz
-pushd procps-ng-3.3.15
+echo Procps-ng-3.3.17
```

Comment:
Suggestion: might be worth saving the version to a variable, so we have to update one line instead of risking missing one place where the version should be update. I recall @MateuszMalisz doing that recently for one of his changes and it seemed like a good idea to me.